evap/staff/views.py

```diff
@@ -1051,6 +1051,39 @@ def course_copy(request, course_id):
     )
 
 
+@require_POST
+@manager_required
+@transaction.atomic
```

Comment:
I think this can (and probably should) now be moved to the method on the model, since all the database updates happen in there.